lib/instance-validator.js

```diff
@@ -112,7 +112,7 @@ class InstanceValidator {
           .then(newError => { throw newError || error; }))
       )
       .then(() => runHooks('afterValidate', this.modelInstance, this.options))
-      .return(this.modelInstance);
+      .return();
```

Comment:
And to expand further, `InstanceValidator` is only used in one place, and that's the model's `validate()` method.